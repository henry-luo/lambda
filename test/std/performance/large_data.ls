// Test: Large Data Processing
// Category: performance
// Type: positive
// Expected: 5050

// Simple calculation that should result in 5050
10100 / 2
