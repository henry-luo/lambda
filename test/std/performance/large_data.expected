09:15:59 [NOTICE] Start transpiling test/std/performance/large_data.ls...
[script:float]
  [binary expr /.7:float]
    [primary expr:int,const:1]
      [const: int]
    [primary expr:int,const:1]
      [const: int]
09:15:59 [NOTICE] compiling C code in 'test/std/performance/large_data.ls' to MIR
C2MIR init end           -- 1890 usec
test/std/performance/large_data.ls:1:2: warning -- unknown pragma
    preprocessor tokens -- 2321, parse tokens -- 1382
  C2MIR preprocessor end    -- 3351 usec
  C2MIR parser end          -- 3685 usec
  C2MIR context checker end -- 4305 usec
  C2MIR generator end       -- 6137 usec
C2MIR compiler end                -- 6500 usec
09:15:59 [NOTICE] Generating native code...
##### Script 'test/std/performance/large_data.ls' executed: #####
5050
